### PR TITLE
Update stack.json

### PR DESCRIPTION
**Pull request by alem355**
annual Update stack Q3

### CODE REVIEW

File: static/stack.json [hunk 1]
```diff
@@ -156,7 +156,7 @@
       "Cases": "Data storage",
       "Research": "",
       "Trial": "<code class='tool'>Cassandra</code> <code class='tool'>MariaDB</code> <code class='platform'>Huawei S3</code>",
-      "Best choice": "<code class='tool'>MongoDB</code> <code class='tool'>ClickHouse</code> <code class='tool'>PostgeSQL</code> <code class='platform'>S3</code> <code class='tool'>Elasticsearch (opendistro)</code>",
+      "Best choice": "<code class='tool'>MongoDB</code> <code class='tool'>ClickHouse</code> <code class='tool'>PostgeSQL</code> <code class='platform'>Yandex S3</code> <code class='tool'>Elasticsearch (opendistro)</code>",
```
Comment (voborgus) [inline static/stack.json:159]: PostgreSQL typo

File: static/stack.json [hunk 2]
```diff
@@ -175,6 +175,14 @@
       "Best choice": "<code class='lang'>TypeScript: NodeJS</code> <code class='lang'>Java: Spring Boot</code> <code class='lang'>Kotlin: Spring Boot</code> <code class='platform'>Platformeco</code>",
       "Hold": "<code class='platform'>ESB</code> <code class='tool'>Debezium / CDC Pattern</code>"
     },
+    {
+      "Area": "Business applications backend",
+      "Cases": "Apisation",
```
Comment (voborgus) [inline static/stack.json:180]: is this term really exists not only in our heads? Is this term is understandable for anyone? Does it mean I should not create an APIs for example with Kotlin?

File: static/stack.json [hunk 3]
```diff
@@ -194,9 +202,9 @@
     {
       "Area": "Frontends",
       "Cases": "Mobile",
-      "Research": "<code class='tool'>Kaspresso</code> <code class='tool'>XCTest</code>",
-      "Trial": "<code class='lang'>React Native</code> <code class='lang'>Kotlin Multiplatform</code>",
-      "Best choice": "<code class='lang'>Swift</code> <code class='tool'>Alamofire</code> <code class='tool'>Core Data</code> <code class='lang'>Kotlin</code> <code class='tool'>Firebase</code> <code class='tool'>Room</code> <code class='tool'>Jetpack Navigation</code> <code class='tool'>Retrofit + Okhttp</code>",
+      "Research": "<code class='tool'>Kaspresso</code> <code class='tool'>XCTest</code> <code class='tool'>Jetpack Compose</code>",
+      "Trial": "<code class='lang'>React Native</code> <code class='lang'>Jetpack Navigation</code>",
```
Comment (voborgus) [inline static/stack.json:206]: JetPack Navigation is a tool

File: static/stack.json [hunk 4]
```diff
@@ -194,9 +202,9 @@
...
+      "Best choice": "<code class='lang'>Swift</code> <code class='tool'>Alamofire</code> <code class='lang'>Kotlin</code> <code class='tool'>Firebase</code> <code class='tool'>Room</code> <code class='tool'>Retrofit + Okhttp</code> <code class='tool'></code>",
```
Comment (voborgus) [inline static/stack.json:207]: mistake here

File: static/stack.json [hunk 5]
```diff
@@ -249,8 +257,8 @@
     },
     {
       "Area": "API management",
-      "Cases": "Gateways & mediation",
-      "Research": "<code class='tool'>Istio</code>",
+      "Cases": "Product to Productintegrations mediators",
```
Comment (voborgus) [inline static/stack.json:260]: typo here

File: static/stack.json [hunk 6]
```diff
@@ -290,8 +298,8 @@
     {
       "Area": "CICD",
       "Cases": "CI tool",
-      "Research": "<code class='tool'>Github Actions</code>",
-      "Trial": "",
+      "Research": "<code class='tool'>Argo workflow</code>",
+      "Trial": "<code class='tool'>Github Actions</code>",
```
Comment (voborgus) [inline static/stack.json:302]: I would say it`s a platform

File: static/stack.json [hunk 7]
```diff
@@ -330,24 +338,24 @@
...
       "Best choice": "<code class='tool'>Ansible</code> <code class='tool'>IaC</code> <code class='tool'>Helm</code> <code class='tool'>Terraform</code>",
-      "Hold": "<code class='tool'>Puppet</code> <code class='tool'>Saltstack</code>"
+      "Hold": "<code class='tool'>Puppet</code>"
     },
     {
       "Area": "CICD",
       "Cases": "Containerization",
       "Research": "",
-      "Trial": "",
+      "Trial": "<code class='platform'>Werf</code>",
```
Comment (voborgus) [inline static/stack.json:350]: is there a mistake to place a `Werf` here? @pavelchalyk

File: static/stack.json [hunk 8]
```diff
@@ -330,24 +338,24 @@
...
       "Best choice": "<code class='platform'>Kubernetes</code> <code class='tool'>Docker</code>",
       "Hold": "<code class='platform'>Openshift</code>"
     },
     {
       "Area": "CICD",
       "Cases": "Secrets Storage",
       "Research": "",
-      "Trial": "",
+      "Trial": "<code class='tool'>GIT Encription</code>",
```
Comment (voborgus) [inline static/stack.json:358]: `GIT Encription` -> `Encrypted in Git`

**Review by voborgus — CHANGES_REQUESTED** [review]

File: static/stack.json (hunk 2 above)
Comment (alem355) [reply static/stack.json:180]: Let's add it for the first time and discuss the next TECH STACK UPDATE MEETING.

File: static/stack.json (hunk 3 above)
Comment (alem355) [reply static/stack.json:206]: done

File: static/stack.json (hunk 5 above)
Comment (alem355) [reply static/stack.json:260]: done

File: static/stack.json (hunk 7 above)
Comment (alem355) [reply static/stack.json:350]: It's not a typo

File: static/stack.json (hunk 1 above)
Comment (alem355) [reply static/stack.json]: Done

**Review by voborgus — APPROVED** [review]

**Pull request merged**